addons/website_event/views/website_event.xml

```diff
@@ -417,7 +417,7 @@
 <template id="registration_attendee_details" name="Registration Attendee Details">
     <div id="modal_attendees_registration" class="modal fade" tabindex="-1" role="dialog">
         <div class="modal-dialog modal-lg">
-            <form id="attendee_registration" t-attf-action="/event/#{slug(event)}/registration/confirm" method="post">
+            <form id="attendee_registration" onsubmit="$(this).find('.btn-primary').button('loading')" t-attf-action="/event/#{slug(event)}/registration/confirm" method="post">
```

Comment:
jquery selector is too lazy it will select all bootstrap primary button on form. remove it.  If it really require handle by js using unique html id